sql-select/ten-actor-first-names.sql

```diff
@@ -0,0 +1,3 @@
+select "firstName"
+from "actors"
+limit 10;
```

Comment:
You're missing a field:

---

![image](https://user-images.githubusercontent.com/16143771/186284349-d2f67c26-672a-4fa1-a1ac-c9100c2d8e94.png)